content/en/docs/score specification/score-spec-reference.md

```diff
@@ -127,21 +127,75 @@ containers:
 
 `variables`: the environment variables for the container. Container variables support both metadata and resource output [placeholders]({{< relref "#placeholder-references" >}}).
 
-`files`: the extra files to mount into the container. Either `content` or `source` must be specified along with `target`.
+`files`: the extra files to mount into the container. Either `content`, `binaryContent` or `source` must be specified.
 
 - `target`: the file path to expose in the container.
-- `mode`: the optional file access mode in octal encoding. For example 0600.
-- `source`: the relative or absolute path to the content file. File content supports both metadata and resource output [placeholders]({{< relref "#placeholder-references" >}}) unless `noExpand` is true.
-- `content`: the inline content for the file. File content supports both metadata and resource output [placeholders]({{< relref "#placeholder-references" >}}) unless `noExpand` is true.
-- `binaryContent`: base64-encoded inline content for the file. This field supports non-utf-8 bytes for binary or archive files. Placeholder expansion is never supported.
-- `noExpand`: if set to true, the placeholders expansion will not occur in the contents of the `content` or `source` file.
+  - `mode`: the optional file access mode in octal encoding. For example 0600.
+  - `source`: the relative or absolute path to the content file. File content supports both metadata and resource output [placeholders]({{< relref "#placeholder-references" >}}) unless `noExpand` is true.
+  - `content`: the inline content for the file. File content supports both metadata and resource output [placeholders]({{< relref "#placeholder-references" >}}) unless `noExpand` is true.
+  - `binaryContent`: base64-encoded inline content for the file. This field supports non-utf-8 bytes for binary or archive files. Placeholder expansion is never supported.
+  - `noExpand`: if set to true, the placeholders expansion will not occur in the contents of the `content` or `source` file.
+
+The list of `files` as an `array` like illustrated below has always been supported:
```

Comment:
Customers will use the first approach they find in the docs. We should describe the map approach here first, and then after that mention that an array based approach is supported but may be deprecated in the future.